### ISSUE TITLE
Is there any way to create directory while downloading blob

### ISSUE THREAD

**Issue by neelam256** [Azure/azure-storage-cpp#334]
I have to create directory in local if it doesn't exist, to download the content from azure blob storage. is there any class /method is available in Azure SDK , @jasonnewyork

**Comment by JinmingHu-MSFT** [comment]
@neelam256 Nope, this c++ sdk only deals with single file download/upload, it cannot recursively sync directory.
